sdk/lib/cmlib/cmlib.h

```diff
@@ -380,7 +380,8 @@ extern ULONG CmlibTraceLevel;
 // Hack since big keys are not yet supported
 //
 #ifdef _BLDR_ 
-#define ASSERT_VALUE_BIG(h, s) DbgPrint("Big keys aren't supported!\n");
+#define ASSERT_VALUE_BIG(h, s) \
+  if (CmpIsKeyValueBig(h,s)) DbgPrint("Big keys aren't supported!\n");
```

Comment:
```suggestion
  do { if (CmpIsKeyValueBig(h,s)) DbgPrint("Big keys aren't supported!\n"); } while (0)
```

Reply:
I'm curious, why?

Reply:
So that the macro can "behave" similarly to a function.
https://stackoverflow.com/questions/257418/do-while-0-what-is-it-good-for
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=define+DPRINT1

Reply:
I see. makes sense